### ISSUE TITLE
absent: ext-treepanel's itemClick Listener

### ISSUE THREAD

**Issue by fabriciomurta** [extnet/Ext.NET#1783]
Found: 7.0.0-preview4

At least this can't be used:

```xml
<ext-treepanel width="500" height="500">
    <listeners>
        <itemclick handler="Ext.toast('item click')" />
    </listeners>
</ext-treepanel>
```

To circumvent the limitation, I can do:
```xml
<ext-treepanel width="500" height="500">
    <viewConfig>
        <ext-treeView>
            <listeners>
                <itemclick handler="Ext.toast('item click')" />
            </listeners>
        </ext-treeView>
    </viewConfig>
</ext-treepanel>
```

According to Ext JS documentation, the [`itemClick` event should be supported](https://docs.sencha.com/extjs/7.2.0/classic/Ext.tree.Panel.html#event-itemclick).

Several other `item*` listeners are also missing from the component, where they should have been supported (and are supported by client-side code).

**Comment by geoffreymcgill** [comment]
It appears the parsers are missing the following pattern in the docs:

```csharp
/**
 * @event itemclick
 * Fires when an item is clicked.
 * @inheritdoc #beforeitemmousedown
 */
```

**Comment by AndreyChechel** [comment]
Fixed in `7.0.0-preview4`
